wasm/wasm_source/src/tx_deactivate_validator.rs

```diff
@@ -0,0 +1,12 @@
+//! A tx to deactivate a validator.
```

Comment:
we could use a single tx type that takes a bool for this, but it's not much difference

Reply:
Ah true, but unless you think there's a nice advantage, I'd rather take the path of least resistance rn and keep it as is.